packages/cli/init/src/utils/loadOpenApiFromUrl.ts

```diff
@@ -0,0 +1,46 @@
+import axios from "axios";
+import { writeFile } from "fs/promises";
+import { dump } from "js-yaml";
+import { join } from "path";
+import tmp from "tmp-promise";
+import { Logger } from "../../../logger/src/Logger";
+
+export type LoadOpenAPIResult = SuccessLoadOpenAPI | FailedLoadOpenAPI;
+
+export enum LoadOpenAPIStatus {
+    Success = "success",
+    Failure = "failure",
+}
```

Comment:
nit: not too many perks of making this an enum (literal is just fine in ts)

Reply:
i was thinking if we have more statuses down the line, it would be a safer way of defining them here

Reply:
for example, if we wanted to break failures down even further ("failed to download", "failed to locate openapi", etc)

Reply:
sounds good